### DISCUSSION TITLE
Riavvio Sonoff T1

### DISCUSSION THREAD

**Discussion by federicomolinaro** [indomus/forum#1623] [HUB: Home Assistant]
Buongiorno, vi chiedo aiuto riguardo ai Sonoff T1. Li ho riprogrammati con il fw Tasmota e li ho installati in tutta la casa sostituendoli agli interruttori tradizionali. Controllo la casa tramite HomeAssistant installato su un raspberry. Il problema è che un paio di sonoff si accendono da soli (e le relative luci che controllano). Prima capitava la notte quando stoppavo tutti i servizi dockerizzati (compreso il servizi mqtt), facevo il backup su NAS e li facevo ripartire. Eliminando tale procedura il problema si presenta ora random, da quello che ho capito quando il wifi per qualche ragione si spegne per qualche secondo. Ho gia impostato PowerOnState = 0 e anche gli altri settaggi mi sembrano corretti. Avete qualche suggerimento su cosa può essere?

**Comment by snakuzzo** [comment]
Non è che per caso hai in retain mqtt lo stato on ?
Dovresti controllare sul broker

**Reply by federicomolinaro** [reply]
Ciao, ancora non sono riuscito a risolvere. E' un interruttore in particolare che si resetta, non ho capito quando ma non dipende dal wifi: fa un beep e poi si accende la luce. Consigli?

**Comment by federicomolinaro** [comment]
Sai indicarmi una guida per verificare sul broker (mosquitto)? Io ho pulito le retain sui client sia via riga di comando:
mosquitto_sub -h <IP> -u <user>-P <password> -t '#' --remove-retained --retained-only
sia tramite Tasmota Device Manager
